packages/color-area/src/ColorArea.ts

```diff
@@ -414,45 +475,69 @@ export class ColorArea extends SpectrumElement {
-                    @input=${this.handleInput}
-                    @change=${this.handleChange}
-                />
-            </div>
+            <fieldset
+                class="fieldset"
+                aria-label=${ifDefined(isMobile ? ariaLabel : undefined)}
+            >
+                <div role="presentation">
+                    <input
+                        type="range"
+                        class="slider"
+                        name="x"
+                        aria-label=${isMobile ? ariaLabelX : ariaLabel}
+                        aria-roledescription=${ariaRoleDescription}
+                        aria-orientation="horizontal"
+                        aria-valuetext=${isMobile
+                            ? ariaValueX
+                            : `${ariaValueX}, ${ariaLabelX}${
+                                  this._valueChanged
+                                      ? ''
+                                      : `, ${ariaValueY}, ${ariaLabelY}`
+                              }`}
+                        min="0"
+                        max="1"
+                        step=${this.step}
+                        tabindex="-1"
+                        .value=${String(this.x)}
+                        @input=${this.handleInput}
+                        @change=${this.handleChange}
+                    />
+                </div>
+                <div role="presentation">
+                    <input
+                        type="range"
+                        class="slider"
+                        name="y"
+                        aria-label=${isMobile ? ariaLabelY : ariaLabel}
+                        aria-roledescription=${ariaRoleDescription}
+                        aria-orientation="vertical"
+                        aria-valuetext=${isMobile
+                            ? ariaValueY
+                            : `${ariaValueY}, ${ariaLabelY}${
+                                  this._valueChanged
+                                      ? ''
+                                      : `, ${ariaValueX}, ${ariaLabelX}`
+                              }`}
+                        orient="vertical"
+                        min="0"
+                        max="1"
+                        step=${this.step}
+                        tabindex="-1"
+                        .value=${String(this.y)}
+                        @input=${this.handleInput}
+                        @change=${this.handleChange}
+                    />
+                </div>
+            </fieldset>
         `;
     }
 
     protected override firstUpdated(changed: PropertyValues): void {
         super.firstUpdated(changed);
         this.boundingClientRect = this.getBoundingClientRect();
 
-        this.addEventListener('focusin', this.handleFocusin);
-        this.addEventListener('focusout', this.handleFocusout);
+        this.addEventListener('focus', this.handleFocus);
+        this.addEventListener('blur', this.handleBlur);
```

Comment:
I do worry about the API name change not being consistent with our other components, though. I think this is going in the right direction, but I feel like it would be good for me to confer with the rest of the team.

Reply:
The function name for these private methods can remain `handleFocusin`/`handleFocusout`, but we should listen for the `focus` and `blur` events in order to set the `focused` state properly.